openapi/paths/organization-exports.yaml

```
@@ -0,0 +1,58 @@
+get:
+  x-products:
+    - Users
+  tags:
+    - Organization data exports
+  summary: Retrieve a list of organization data exports.
+  operationId: GetOrganizationExportCollection
+  x-sdk-operation-name: getAll
+  description: |
+    Retrieve a list of organization data exports.
+  parameters:
+    - $ref: ../components/parameters/collectionLimit.yaml
+    - $ref: ../components/parameters/collectionOffset.yaml
+    - $ref: ../components/parameters/collectionFilter.yaml
+    - $ref: ../components/parameters/collectionQuery.yaml
+  responses:
+    '200':
+      description: A list of organization data exports was retrieved successfully.
```

nitpick: avoid past tense (where possible)
```suggestion
      description: A list of organization data exports is successfully retrieved.
```